apps/etterlatte-migrering/src/main/kotlin/migrering/verifisering/Verifiserer.kt

```diff
@@ -80,10 +80,14 @@ internal class Verifiserer(
                     val foedselsdato: LocalDate =
                         person.getOrNull()?.foedselsdato?.verdi
                             ?: request.soeker.getBirthDate()
-                    if (foedselsdato.isBefore(LocalDate.of(2006, Month.JANUARY, 1)) && !request.dodAvYrkesskade) {
+                    if (foedselsdato.isBefore(LocalDate.of(2005, Month.DECEMBER, 1)) && !request.dodAvYrkesskade) {
                         logger.warn("Søker er over 18 år og det er ikke yrkesskade")
                         return listOf(SoekerErOver18)
                     }
+
+                    if (person.getOrNull()?.doedsdato != null) {
```

Comment:
Desse sakene er ikkje nødvendigvis løpande i Pesys, så vi bør sjekke at søkar framleis er i live - for migreringar fram til no har Pesys-uttrekket (så godt som) sikra det for oss.